src/test/datascience/widgets/thirdpartyWidgets.vscode.common.test.ts

```diff
@@ -0,0 +1,230 @@
+        await executeCellAndWaitForOutput(cell1, comms);
+        await assertOutputContainsHtml(cell0, comms, ['Click Me!', '<button']);
+
+        // Click the button and verify we have output in the same cell.
+        await clickWidget(comms, cell0, 'button');
+        await assertOutputContainsHtml(cell0, comms, ['>Figure 1<', '<canvas', 'Download plot']);
+    });
+    test('Render IPySheets', async function () {
+        // https://github.com/microsoft/vscode-jupyter/issues/10506
+        return this.skip();
+        const comms = await initializeNotebookForWidgetTest(api, disposables, {
+            templateFile: 'ipySheet_widgets.ipynb'
+        });
+        const [, cell1] = vscodeNotebook.activeNotebookEditor!.notebook.getCells();
+
+        await executeCellAndWaitForOutput(cell1, comms);
+        await assertOutputContainsHtml(cell1, comms, ['Hello', 'World', '42.000']);
+    });
+    test('Render IPySheets & search', async function () {
+        // https://github.com/microsoft/vscode-jupyter/issues/10506
+        return this.skip();
+        const comms = await initializeNotebookForWidgetTest(api, disposables, {
+            templateFile: 'ipySheet_widgets_search.ipynb'
+        });
+        const [, cell1, cell2] = vscodeNotebook.activeNotebookEditor!.notebook.getCells();
+
+        await executeCellAndWaitForOutput(cell1, comms);
+        await executeCellAndWaitForOutput(cell2, comms);
+        await assertOutputContainsHtml(cell1, comms, ['title="Search:"', '<input type="text']);
+        await assertOutputContainsHtml(cell2, comms, ['>train<', '>foo<']);
+
+        // Update the textbox widget.
+        await comms.setValue(cell1, '.widget-text input', 'train');
+        await assertOutputContainsHtml(cell2, comms, ['class="htSearchResult">train<']);
+    });
+    test('Render IPySheets & slider', async function () {
+        // https://github.com/microsoft/vscode-jupyter/issues/10506
+        return this.skip();
+        const comms = await initializeNotebookForWidgetTest(api, disposables, {
+            templateFile: 'ipySheet_widgets_slider.ipynb'
+        });
+        const [, cell1, cell2, cell3] = vscodeNotebook.activeNotebookEditor!.notebook.getCells();
+
+        await executeCellAndWaitForOutput(cell1, comms);
+        await executeCellAndWaitForOutput(cell2, comms);
+        await executeCellAndWaitForOutput(cell3, comms);
+        await assertOutputContainsHtml(cell1, comms, ['Continuous Slider']);
+        await assertOutputContainsHtml(cell2, comms, ['Continuous Text', '<input type="number']);
+        await assertOutputContainsHtml(cell3, comms, ['Continuous Slider', '>0<', '>123.00']);
+
+        // Update the textbox widget (for slider).
+        await comms.setValue(cell2, '.widget-text input', '5255');
+        await assertOutputContainsHtml(cell3, comms, ['>5255<', '>5378.0']);
+    });
+    test.skip('Render ipyvolume (slider, color picker, figure)', async function () {
```

Comment:
How come this one is skipped?

Reply:
Was already skipped, will need to figure out why it wasn't working.